config/routes.rb

```diff
@@ -1,25 +1,63 @@
-Rails.application.routes.draw do
+Rails.application.routes.draw do  
+  # PART 1: EACH IN ERB
+  # ===================
 
-  get("/", { :controller => "fortunes", :action => "lucky"})
+  # Let users visit URLs:
+  # - /lottery/lucky
+  # - /lottery/unlucky
 
-  get("/lucky-numbers", { :controller => "fortunes", :action => "lucky"})
-  get("/unlucky-numbers", { :controller => "fortunes", :action => "unlucky"})
-
-  # PART 1: STATIC ROUTES
-  # =====================
+  # PART 2: RCAV DEBUGGING
+  # ======================
 
   # Uncomment each route below ONE AT A TIME and debug.
+  # Do NOT uncomment more than one at a time, or you'll be dealing with multiple syntax errors at once.
+
+  # get("/zodiacs/aries", { :controller =>  fire, action =>  "ram" })
+  # get("/zodiacs/leo", { :controller => "fire", :action => "lion" })
```

Comment:
Woah wait, what it always the case that `Fire_controller` will still match when you just specify `:controller => "fire"` in a Route? 

![bump](https://user-images.githubusercontent.com/17581658/88100551-4a2e4000-cb62-11ea-862b-b1cabf4576c8.png)

Apparently it _does_ match now, which I think seeing this capital letter might be confusing since it doesn't apparently cause issues?

Reply:
@raghubetina

Reply:
@jelaniwoods It definitely wasn't always the case. We should plant some other controller filename mismatch bug, then; I think that was the only one.

Reply:
Good catch.

Reply:
Something weird is going on... it feels like VSCode is downcasing the filename or something.